### ISSUE TITLE
How to billboard lines towards the camera?

### ISSUE THREAD

**Issue by invokermain** [james-j-obrien/bevy_vector_shapes#11]
Hey,

Great library, really intuitive. There's only one part I don't understand.

I can see that lines are treated differently. `Note that lines rotate around their direction while all other shapes will fully face the camera at all times.`. What are the implications of this as a user though?

I'm trying to draw a bunch of little x's that face the camera at all times.

**Comment by invokermain** [comment]
Rectangles as lines makes sense, thanks for the quick reply! Also I think I understand, e.g. in my app https://invokermain.github.io/spacesim/ when I replace the axes stemming out from the Sun it would be weird for those to be fixed orientated towards the camera.

Also just want to say really nice library again, was refreshing to be able to go from finding the library to having exactly what I wanted rendered in less than an hour.

**Comment by invokermain** [comment]
Hmm maybe billboarding doesn't do what I think it does, but it has some weird behaviour compared to the example see here: https://gyazo.com/c2d1271a3a1b1a2b869e23e85b508eba

**Comment by james-j-obrien** [comment]
That definitely doesn't look correct, unfortunately I'm on vacation at the moment and so can't test locally, if you can send me the code that you're using to create that shape I'll take a look when I'm back home this weekend.

**Comment by invokermain** [comment]
Enjoy the vacation 🌴 I put together a simple example that shows a collection of weird behaviour.

```rust
use bevy::prelude::*;
use bevy_vector_shapes::prelude::*;

mod gallery_3d;

fn main() {
    App::new()
        .add_plugins(DefaultPlugins)
        .add_plugin(ShapePlugin {
            base_config: ShapeConfig {
                alignment: Alignment::Billboard,
                ..ShapeConfig::default_3d()
            },
            ..default()
        })
        .insert_resource(ClearColor(Color::DARK_GRAY))
        .insert_resource(Msaa::Off)
        .add_startup_system(setup)
        .add_system(draw_gallery)
        .run();
}

fn setup(mut commands: Commands) {
    commands.spawn(Camera3dBundle::default());
}

fn draw_gallery(
    time: Res<Time>,
    mut painter: ShapePainter,
    mut cameras: Query<&mut Transform, With<Camera3d>>,
) {
    let mut camera = cameras.single_mut();

    // Camera flies in a circle on the YZ plane always looking at the origin
    *camera = Transform::from_translation(
        Quat::from_rotation_x(time.elapsed_seconds()) * Vec3::new(0., 0., 10.),
    )
    .looking_at(
        Vec3::ZERO,
        if camera.translation.z < 0.0 {
            Vec3::Y
        } else {
            -1.0 * Vec3::Y
        },
    );

    // Spins in a circle
    painter.translate(Vec3::X);
    painter.rect(Vec2::new(0.25, 1.0));

    // Works as expected
    painter.translate(Vec3::ZERO);
    painter.rect(Vec2::new(0.25, 1.0));

    // Doesn't show at all
    painter.translate(Vec3::X * -1.0);
    painter.rect(Vec2::new(0.25, 1.0));
}
```

**Comment by james-j-obrien** [comment]
Thanks for the example code, I've pushed a fix to main that should resolve the spinning you were experiencing there. 

The final rectangle is actually there it's placed on top of the first one, you are using translate to set the translation rather than set_translation. With this code here and the version of the library on main you should see expected behavior for that example: 
```rust
    // Spins in a circle
    painter.set_translation(Vec3::NEG_X);
    painter.rect(Vec2::new(0.25, 1.0));

    // Works as expected
    painter.set_translation(Vec3::ZERO);
    painter.rect(Vec2::new(0.25, 1.0));

    // Doesn't show at all
    painter.set_translation(Vec3::X);
    painter.rect(Vec2::new(0.25, 1.0));
```

Thanks again for the assistance here, I should be releasing 0.4.6 with a few bug fixes in the next day or two alongside 0.5.0 supporting bevy 0.11.

Let me know if you have any other behavior which seem suspect.
